.github/workflows/ci.yml

```diff
@@ -88,6 +88,43 @@ jobs:
             - name: Run tests
               run: vendor/bin/simple-phpunit
 
+    phpunit-dev:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php-versions: ['8.0']
+            fail-fast: false
+        name: PHP ${{ matrix.php-versions }} Test dev dependencies on ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-versions }}
+                  extensions: zip
+
+            - name: Get composer cache directory
+              id: composercache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composercache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+                  restore-keys: ${{ runner.os }}-composer-
+
+            - name: Allow dev dependencies
+              run: composer config minimum-stability dev
```

Comment:
Normally we have `minimum-stability: "dev"` in `composer.json` by default... and that's what we always use. If we did that, then we wouldn't need this second job, I think. For example: https://github.com/symfony/string/blob/d70c35bb20bbca71fc4ab7921e3c6bda1a82a60c/composer.json#L39

Reply:
A little more info. What we normally do is:

* Put `"minimum-stability": "dev"` directly in `composer.json`
* For "normal" builds (e.g. released versions like Symfony 5.3), we use `--prefer-stable` with  `composer update`.
* For non-released builds (e.g. Symfony 6), we would omit the `--prefer-stable`. 

BUT, I think these are not-important details. The current builds DO test the correct stuff. I vote we merge this, and then we could potentially optimize this build file later to reduce duplication.

Reply:
I think it would be nice to have a better testing of different versions, as you said it can be done on another Pr. I can bring some of the workflows we use at Sonata if you dont have a standard way for all sf repositories.